client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -354,6 +356,80 @@ class EmailProvidersComparison extends Component {
 		);
 	}
 
+	getAvailableDiscountForGoogle(
+		currencyCode,
+		isEligibleForFreeTrial,
+		gSuiteProduct,
+		productIsDiscounted,
+		standardPrice
+	) {
+		const { translate } = this.props;
+		if ( productIsDiscounted ) {
+			return (
+				<span className="email-providers-comparison__discount-with-renewal">
+					{ translate(
+						'%(discount)d%% off{{span}}, %(discountedPrice)s billed today, renews at %(standardPrice)s{{/span}}',
```

Comment:
When I enable a sale coupon, it shows the following:

![image](https://user-images.githubusercontent.com/594356/155364447-def3c708-48b3-4d35-bd2c-7bcec8380d6f.png)

It's all good except for the extra space before the fist comma, which has something to do with this `<span>`.